modules/web/src/app/wizard/step/cluster/template.html

```diff
@@ -113,8 +113,23 @@
               </mat-option>
             </mat-select>
           </mat-form-field>
+          <mat-form-field class="km-dropdown-with-suffix">
+            <mat-label>Expose Strategy</mat-label>
+            <mat-select [formControlName]="Controls.ExposeStrategy"
+                        class="km-select-ellipsis"
+                        disableOptionCentering>
+              <mat-option *ngFor="let exposeStrategy of exposeStrategies"
+                          [value]="exposeStrategy">
+                {{exposeStrategy}}
+              </mat-option>
+            </mat-select>
+            <mat-hint>Leave empty to use the default value. Check the documentation for more information&nbsp;<a target="_blank"
+                 rel="noopener"
+                 href="https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/networking/expose-strategies/#configure-the-expose-strategy">here</a>.
+            </mat-hint>
+          </mat-form-field>
           <div fxLayout="row"
-               fxLayoutGap="50px"
+               fxLayoutGap="100px"
```

Comment:
I think `100px` is too much space or is there a specific reason to increase this?

Reply:
It's to facilitate the `mat-hint` 50px gap was too low for it. Although we can use padding as an alternate as well.